controllers/controllers/networking/cfdomain_controller_test.go

```diff
@@ -113,6 +114,10 @@ var _ = Describe("CFDomainReconciler Integration Tests", func() {
 			}).Should(Succeed())
 		})
 	})
+
```

Comment:
This log message is a result of the logger from `log := shared.ObjectLogger(r.log, cfDomain)` in the `ReconcileResource` method. There's no chance of that method not producing a logger, so I don't think we need to test this log.

The log which could potentially be discarded is in the `finalizeCFDomain()` method. So if you move this test up into the delete 'when' block and check for the 'finalizer removed' message, this would be better.